discoverable/purplebooth.md

```diff
@@ -0,0 +1,89 @@
+```markdown
+# Project Title
+
+One Paragraph of project description goes here
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
```

Comment:
These are the guidelines that will be used in the project which will be commenced by me in a few week's time!